docs/source/advanced_tutorial/index.rst

```diff
@@ -0,0 +1,17 @@
+Advanced Tutorial
+#################
+
+This advanced tutorial describes the process for developing residential measures on a branch of the `OpenStudio-BEopt <https://github.com/NREL/OpenStudio-BEopt>`_ repository, and subsequently pulling them into the ResStock workflow for creating and testing residential building models.
```

Comment:
Need some context for why you would want to do this. For example, it would be only if you want to customize any of the existing residential modeling algorithms or add a new technology model, correct?

Reply:
I believe so.